cleanup-imported-ebs-snapshots/cloudformation.yaml

```diff
@@ -48,7 +69,7 @@ Resources:
     DependsOn: LambdaLogGroup
     Properties:
       FunctionName: elastio-cleanup-imported-ebs-snapshots
-      Description: 'A Lambda function to delete EBS snapshots tagged with elastio:imported-to-rp'
+      Description: "A Lambda function to delete EBS snapshots tagged with elastio:imported-to-rp"
```

Comment:
Here again you're referring to the tag name which might actually be different.  I think you should remove the parameter that allows overriding the tag name.  It's not a requirement today.